content/blog/oddnews/oddnews-21.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CSS Cascade Layers
+issue: 21
+url: http://eepurl.com/i_NTYw
+date: 2025-03-13
+tags:
+  - CSS
+  - Anchor Positioning
+  - Typography
+length: 3 mins
+image:
+  src: winging-it/winging-it-1.jpg
+summary: |
+  Here’s what we’re up to:
+  - Revisiting Fluid Type on Winging It with Richard Rutter | March 20, 2025
+  - New features for OddContrast
+  - CSS Layout Workshop coming in April 2025
+  - Custom CSS Functions in the Browser article by Miriam Suzanne
+---
+
+{% import "utility.macros.njk" as utility %}
+
+{{ utility.main_action(
+  'Subscribe to OddNews »',
+  subscribe_url
+) }}
```

Comment:
I see we're using this pattern on all winging-it post, probably so we can get access to the `subscribe_url`? I think we could be using the built-in actions from yaml if we prefer. It would be a bit more consistent with other posts:

```yaml
eleventyComputed:
  action:
    text: Subscribe to OddNews »
    url: '{{ subscribe_url }}'
```